students/mkdir01/lesson06/test_mailroom.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+from io import StringIO
+import sys
+from contextlib import redirect_stdout
+import pytest
+
+
+# test input_name():
+from  mailroom3 import input_name
+def test_1(monkeypatch):  # tests "list"
+    monkeypatch.setattr('builtins.input', lambda x: "list")
+    f = StringIO()
+    with redirect_stdout(f):
+        input_name()
+    testdata = f.getvalue()
+    assert testdata == "\n\nJohn Doe\nJane Doe\nJohn Smith\nJane Smith\nBilly Jo Jones\n\n\n"
+
+def test_2(monkeypatch):  # tests ""
+    monkeypatch.setattr('builtins.input', lambda x: "")
+    testdata = input_name()
+    assert testdata == None
+
+def test_3(monkeypatch):  # tests name
+    monkeypatch.setattr('builtins.input', lambda x: "John")
+    testdata = input_name()
+    assert testdata == "John"
+
+
+#test input_dollars():
+from  mailroom3 import input_dollars
+def test_4(monkeypatch):  # tests name from dict with dollars > 0
+    monkeypatch.setattr('builtins.input', lambda x: "500")
+    testdata = input_dollars('John Doe')
+    assert testdata == 1
+
+def test_5(monkeypatch):  # tests name from dict with value error
```

Comment:
your test names should have meaningful names like;
`def test_raises_on_invalid_dollars`